ui/src/app/store/auth/effects.ts

```diff
@@ -116,10 +124,16 @@ export class AuthEffects {
   @Effect()
   public LogOut: Observable<any> = this.actions.pipe(
     ofType(logOut),
-    map(() => {
-      this.authService.removeToken();
+    switchMap(() => {
       this.router.navigateByUrl(ROUTERS_URL.LOGIN);
-      return resetUserInfo();
+
+      return merge(of(clearUserToken()), of(resetUserInfo()));
```

Comment:
no need to merge, can return an array of actions

Reply:
Tried like that, it doesn't work